mmdet/apis/train.py

```diff
@@ -68,6 +68,67 @@ def set_random_seed(seed, deterministic=False):
         torch.backends.cudnn.benchmark = False
 
 
+def auto_scale_lr(cfg, distributed, logger):
```

Comment:
We can delete some blank lines in this function to make this function compact.

Reply:
Done